### PR TITLE
Styles (<LoadSpinner/>): As a user, while the business list loads, I want to be able to see a spinning loading animation styled to align with the overall app palette.

### PR DESCRIPTION
**Pull request by CodeMeKathy**
## Description
This Pull Request (PR) modifies the `<LoadSpinner/>` stylings of the <LoadSpinner/> to align with the overall app palette.

## Acceptance Criteria
- [✅] Adjust the stylings of the <LoadSpinner/> to align with the application palette. 

## Related Issue/Ticket
N/A

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix(es)              |
|     | :sparkles: New feature     |
| ✅    | :hammer: Refactoring       |
|     | :100: Add tests            |
|   | :link: Update dependencies |
|     | :scroll: Docs              |

## UI Updates

<details>
<summary>Business List Page UI Updates</summary>

- ## Updated LoadSpinner Styles
_* Non-Auto Loop GIFs included_

|   Previous UI  | PR Update                       |
| --- | -------------------------- |
| Before_loadSpinner | After_loadSpinner

### CODE REVIEW

File: client/src/components/LoadSpinner/LoadSpinner.css [hunk 1]
```diff
@@ -1,95 +1,117 @@
...
 }
-.ldio-cs9r5gg8z0w div:nth-child(11) {
+.LoadSpinner-Wrapper div:nth-child(11) {
   transform: rotate(300deg);
   animation-delay: -0.08333333333333333s;
-  background: #456caa;
+  background: #f32dc8;
 }
-.ldio-cs9r5gg8z0w div:nth-child(12) {
+.LoadSpinner-Wrapper div:nth-child(12) {
```
Comment (cynthiaeddy) [inline client/src/components/LoadSpinner/LoadSpinner.css:75]: Thanks for changing the name for clarification!

**Review by cynthiaeddy — APPROVED** [review]
Nicely done!

**Review by sergiobtos — APPROVED** [review]
Good.

File: client/src/components/LoadSpinner/LoadSpinner.css [hunk 2]
```diff
@@ -1,95 +1,117 @@
...
-  transform-origin: 6px 52px;
-}
-.ldio-cs9r5gg8z0w div:nth-child(1) {
+  animation: LoadSpinner-Wrapper linear 1s infinite;
+  background: #f32dc8;
+  width: 11.82px;
+  height: 23.64px;
+  border-radius: 5.91px / 11.82px;
+  transform-origin: 5.91px 51.22px;
+}
```
[the remark this answers is not in the document]
Comment (CodeMeKathy) [reply client/src/components/LoadSpinner/LoadSpinner.css:20]: Given that it is an auto-generated loader, and as you indicated, the difference should be minimal, I would air on the side of caution and not manipulate it. Thanks for the food for thought!

**Pull request merged**